doc/examples/plot_inpaint.py

```diff
@@ -0,0 +1,58 @@
+"""
+===========
+Inpainting
+===========
+Inpainting [1]_ is the process of reconstructing lost or deteriorated
```

Comment:
I believe you need to put the references in the same docstring, based on the error on Travis.